lib/discordrb.rb

```diff
@@ -35,6 +35,18 @@ module Discordrb
     direct_message_typing: 1 << 14
   }.freeze
 
+
+  COMMAND_CONTEXTS = {
```

Comment:
Looks good for the most part, but it may be better to have these two hash maps defined in [lib/discordrb/data/interaction.rb](https://github.com/shardlab/discordrb/blob/main/lib/discordrb/data/interaction.rb)